pkg/fuzztesting/.gitignore

```diff
@@ -0,0 +1 @@
+fuzztests/testdata/fuzz/FuzzAllMappers/
```

Comment:
Can we add this to the existing [.gitignore](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/.gitignore) for better maintainability in the future?